src/locale/messages.fr.xlf

```diff
@@ -85,6 +85,25 @@
           <context context-type="sourcefile">src/app/shared/guards/pending-changes-guard.ts</context>
           <context context-type="linenumber">38</context>
         </context-group>
+      </trans-unit><trans-unit id="3e7d68a4322602de77d9ddbc73b29aa31e65dc41" datatype="html">
+        <source>Advanced parameters</source><target state="new">Advanced parameters</target>
```

Comment:
Please translate them as you speak French :-) (you can use PoEdit for instance)

Reply:
fixed